# Header header
int16 frame
msg_kitti_object[] objects